src/network/objectracker.py

```diff
@@ -1,7 +1,26 @@
+"""
+src/network/objectracker.py
+===========================
+
+addr sending -> per node upload queue, and flush every minute or so
+inv sending -> if not in bloom, inv immediately, otherwise put into a per node upload queue and flush every
+minute or so
+data sending -> a simple queue
+
+no bloom
+- if inv arrives
+  - if we don't have it, add tracking and download queue
+  - if we do have it, remove from tracking
+tracking downloads
+- per node hash of items the node has but we don't
+tracking inv
+- per node hash of items that neither the remote node nor we have
+
+"""
```

Comment:
These are more like my notes from the design phase, I'm not sure they are accurate, let's add a todo or fixme or something here so that I can take a look at it again.